src/panoptes/pocs/camera/camera.py

```diff
@@ -875,11 +876,10 @@ def _setup_observation(self, observation, headers, filename, **kwargs):
                                       f' {observation.filter_name}: {e!r}')
                     raise (e)
 
-            else:
-                self.logger.info(f'Filter {observation.filter_name} requested by'
-                                 f' observation but {self.filterwheel} is missing that filter, '
-                                 f'using'
-                                 f' {self.filter_type}.')
+            elif not observation.dark:
+                self.logger.warning(f'Filter {observation.filter_name} requested by'
+                                    f' observation but {self.filterwheel} is missing that filter, '
+                                    f'using {self.filter_type}.')
```

Comment:
The logic of this and the message isn't clear to me here.

Reply:
Given the changes to add `observation.image_type` below, this would just become something like:

```py
elif observation.image_type != ImageType.DARK
```

Or just `"dark"` if being lazy. Then the `dark` param above is not needed.

Reply:
I think the `dark` attribute *is* needed. It does not correspond to image type, but tells the camera that the exposure should be taken with the shutter closed or with the FW in the dark position (i.e. for darks *or* bias frames).